sdk/src/loom/ModestMaps/core/painter/TilePool.ls

```diff
@@ -14,8 +14,8 @@ package loom.modestmaps.core.painter
      */ 
     public class TilePool 
     {
-        protected static const MIN_POOL_SIZE:int = 256;
-        protected static const MAX_NEW_TILES:int = 256;
+        protected static const MIN_POOL_SIZE:int = 128;
+        protected static const MAX_NEW_TILES:int = 1024;
```

Comment:
I just tested it and even 512 tiles seem to be enough for it to only alloc once per session.